Makefile

```diff
@@ -5,14 +5,27 @@ ifdef DEPLOYMENT
 endif
 
 DOCKER_COMPOSE = docker-compose -f docker-compose.yml
-DOCKER_BUILD_CMD = docker-compose build $(BUNDLE_FLAGS)
 
 ifndef JENKINS_URL
-	DOCKER_COMPOSE += -f docker-compose.development.yml
+  ifndef ON_CONCOURSE
+    DOCKER_COMPOSE += -f docker-compose.development.yml
+  endif
```

Comment:
Should these be tabs?

Reply:
only needs to be tabs when inside rules.

Generally for nesting with definitions, you can use spaces.

Can change this to tabs if you prefer